### PR TITLE
MARP-560 Introduce new fields - names and descriptions- support multilingualism

### PR DESCRIPTION
**Pull request by phhung-axonivy**
Hi @ivy-rew 
We are implementing the feature to support multilingualism for the landing page on the new marketplace's website.
We want to introduce two new fields such as names, and descriptions to display the translated text.
E.g for the description of the Axon Ivy portal
image
image

Can you help me review this pull request and give me your feedback?

### CODE REVIEW

File: market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java [hunk 1]
```diff
@@ -23,9 +23,31 @@ public class MarketMetaSchema {
   public String version;
   @NotNull @Size(min = 4, max = 24)
   public String name;
+  public static class DisplayName {
+    @NotNull
+    @Size(min = 2)
+    public String locale;
```
Comment (ivy-rew) [inline market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java:29]: I'd give examples here; `@Examples({'de', 'en'})` so if one edits these product descriptions with a real editor there will be auto-completion support.

**Review by ivy-rew — APPROVED** [review]
cool :+1:

File: market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java [hunk 2]
```diff
@@ -23,9 +23,31 @@ public class MarketMetaSchema {
...
+  }
+  public List<DisplayName> names;
+
   @NotNull @Size(min = 5, max = 200)
   public String description;
 
+  public static class DisplayDescription {
+    @NotNull
+    @Size(min = 2)
+    public String locale;
```
Comment (ivy-rew) [inline market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java:43]: examples?

File: market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java (hunk 1 above)
Comment (phhung-axonivy) [reply market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java:30]: I added example for it.

File: market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java (hunk 2 above)
Comment (phhung-axonivy) [reply market-test/src/main/java/com/axonivy/market/MarketMetaSchema.java:45]: I added example for it.

**Pull request merged**